src/api/offerings.js

```diff
@@ -29,6 +29,51 @@ export const getOfferingData = async (offeringId) => {
   return null
 }
 
+/// ////////////////       STUDENT OFFERING FUNCTIONS       ////////////////////
+
+/**
+ * Gets the data for an offering from the CT API if the student is authenticated
+ * @returns The offering data
+ */
+export const getOfferingsByStudent = async () => {
+  if (!isUserAuthenticated()) return null
+
+  const url = format(ENDPOINTS.OFFERING, 'ByStudent')
```

Comment:
Since this is just a constant endpoint, we don't need format, you can just add this as an entry in ENDPOINTS. the ENDPOINTS.OFFERING is for formatting an offeringID into the request URL